optimum/quanto/models/diffusers_models.py

```diff
@@ -115,65 +117,67 @@ def quantize(
         return cls(model)
 
     @classmethod
-    def from_pretrained(cls, model_name_or_path: Union[str, os.PathLike]):
+    def from_pretrained(cls, pretrained_model_name_or_path: Union[str, os.PathLike], **kwargs):
```

Comment:
I think this should be a relatively easy breaking change. I can revert this back to `model_name_or_path` but I mainly did it to harmonize with the rest of the libraries.